python/setup.py.in

```diff
@@ -165,9 +165,9 @@ if '${WITH_MKL}' == 'ON':
     shutil.copy('${MKLML_LIB}', libs_path)
     shutil.copy('${MKLML_IOMP_LIB}', libs_path)
     package_data['paddle.libs']+=['libmklml_intel' + ext_name,'libiomp5' + ext_name]
-if '${CMAKE_BUILD_TYPE}' == 'Release':
-    # only change rpath in Release mode.
-    if '${WITH_MKLDNN}' == 'ON':
+if '${WITH_MKLDNN}' == 'ON':
```

Comment:
I though this could be more clear, since actually ngraph depend on mkldnn, and will do nothing with debug mode. 
```
if '${CMAKE_BUILD_TYPE}' == 'Release':
    if '${WITH_MKLDNN}' == 'ON':
        ...
    if '${WITH_NGRAPH}' == 'ON':
        ....
```

Reply:
Let's look into simplifying this.